tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -16,22 +16,24 @@ import { PackageJson, TsConfig } from '../../types';
 import generator from './index';
 import { MigrateConvergedPkgGeneratorSchema } from './schema';
 
+interface AssertedSchema extends MigrateConvergedPkgGeneratorSchema {
+  name: string;
+}
+
 describe('migrate-converged-pkg generator', () => {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   const noop = () => {};
-  const originalConsoleLog = {
-    log: console.log,
-    warn: console.warn,
-  };
-  console.log = noop;
-  console.warn = noop;
 
   let tree: Tree;
-  const options: MigrateConvergedPkgGeneratorSchema = { name: '@proj/react-dummy' };
+  const options = { name: '@proj/react-dummy' };
 
   beforeEach(() => {
     jest.restoreAllMocks();
 
+    jest.spyOn(console, 'log').mockImplementation(noop);
+    jest.spyOn(console, 'info').mockImplementation(noop);
+    jest.spyOn(console, 'warn').mockImplementation(noop);
```

Comment:
Q: Why did we need this again?

Reply:
to not spam logs when running this spec. nx logger uses `console` under the hood. 

In components this is unfortunately handled on global level, going forward I'd recommend to be explicit about turning off native platform logs etc

Reply:
also jest doesn't mock method it spies on (jasmine does for example...)